operate/client/src/App/ProcessInstance/ProcessInstanceHeader/index.tsx

```diff
@@ -23,59 +25,80 @@ import {InstanceHeader} from 'modules/components/InstanceHeader';
 import {Skeleton} from 'modules/components/InstanceHeader/Skeleton';
 import {notificationsStore} from 'modules/stores/notifications';
 import {authenticationStore} from 'modules/stores/authentication';
+import {processStore} from 'modules/stores/process';
+import {IS_VERSION_TAG_ENABLED} from 'modules/feature-flags';
+import {VersionTag} from './styled';
 
-const getHeaderColumns = (isMultiTenancyEnabled: boolean = false) => {
-  return [
-    {
-      name: 'Process Name',
-      skeletonWidth: '94px',
-    },
-    {
-      name: 'Process Instance Key',
-      skeletonWidth: '136px',
-    },
-    {
-      name: 'Version',
-      skeletonWidth: '34px',
-    },
-    ...(isMultiTenancyEnabled
-      ? [
-          {
-            name: 'Tenant',
-            skeletonWidth: '34px',
-          },
-        ]
-      : []),
+const headerColumns = [
+  'Process Name',
+  'Process Instance Key',
+  'Version',
+  'Version Tag',
+  'Tenant',
+  'Start Date',
+  'End Date',
+  'Parent Process Instance Key',
+  'Called Process Instances',
+] as const;
 
-    {
-      name: 'Start Date',
-      skeletonWidth: '142px',
-    },
-    {
-      name: 'End Date',
-      skeletonWidth: '142px',
-    },
-    {
-      name: 'Parent Process Instance Key',
-      skeletonWidth: '142px',
-    },
-    {
-      name: 'Called Process Instances',
-      skeletonWidth: '142px',
-    },
-  ];
-};
+const skeletonColumns: {
+  name: (typeof headerColumns)[number];
+  skeletonWidth: string;
+}[] = [
+  {
+    name: 'Process Name',
+    skeletonWidth: '94px',
+  },
+  {
+    name: 'Process Instance Key',
+    skeletonWidth: '136px',
+  },
+  {
+    name: 'Version',
+    skeletonWidth: '34px',
+  },
+  {
+    name: 'Start Date',
+    skeletonWidth: '142px',
+  },
+  {
+    name: 'End Date',
+    skeletonWidth: '142px',
+  },
+  {
+    name: 'Parent Process Instance Key',
+    skeletonWidth: '142px',
+  },
+  {
+    name: 'Called Process Instances',
+    skeletonWidth: '142px',
+  },
+] as const;
```

Comment:
I tried to make this a bit simpler. Please let me know what you think or if you have some ideas to further improve it.

Reply:
I think it is a bit simpler with the changes, not sure if we can do much more. The structure itself is a bit confusing, but this looks good to me.